src/selectors/entities/teams.js

```diff
@@ -158,7 +158,7 @@ export const getJoinableTeams = createSelector(
             openTeams[id] = teams[id];
         }
 
-        return openTeams;
+        return Object.values(openTeams);
```

Comment:
Instead of constructing an object and then using `Object.values` on it, you could replace the body of this with a `.map` call to simplify it a bit and skip creating the intermediate object
```
(teams, joinableTeamIds) -> {
    return joinableTeamIds.map((id) => teams[id]);
}
```